### ISSUE TITLE
face recognition

### ISSUE THREAD

**Issue by omis76** [boyw165/my-dlib-experiment#2]
have you tried dlib_face_recognition_resnet_model_v1 this model to get 128 vector dimension to perform face recognition?

**Comment by tcw165** [comment]
Nope. Sorry for the late reply because I am busy of company tasks.

**Comment by wonders-gs** [comment]
hope you can start face recognition  ,thanks
